app/javascript/src/components/Dashboard/Contacts/Table/utils.js

```diff
@@ -0,0 +1,15 @@
+import dayjs from "lib/dayjs";
```

Comment:
Go through this section in Miscellaneous tips and fix this import: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F

Reply:
Fixed with #31